quartz/plugins/transformers/ofm.ts

```diff
@@ -105,7 +105,7 @@ export const arrowRegex = new RegExp(/(-{1,2}>|={1,2}>|<-{1,2}|<={1,2})/, "g")
 // (#[^\[\]\|\#]+)?  -> # then one or more non-special characters (heading link)
 // (\|[^\[\]\#]+)? -> \| then one or more non-special characters (alias)
 export const wikilinkRegex = new RegExp(
-  /!?\[\[([^\[\]\|\#\\]+)?(#+[^\[\]\|\#\\]+)?(\\?\|[^\[\]\#]+)?\]\]/,
+  /(\|[^\|\n]*)?!?\[\[([^\[\]\|\#\\]+)?(#+[^\[\]\|\#\\]+)?(\\?\|[^\[\]\#]+)?\]\]([^\|\n]*\|)?/,
```

Comment:
can we update the comment explaining this?

Reply:
I have updated the comment with an explanation.